assets/scripts/lobby/options.js

```diff
@@ -275,28 +275,45 @@ var userOptions = {
         }
     },
 
-   optionProposedTeamIcon: {
+   optionDisplayProposedTeamIcon: {
```

Comment:
Just occured to me that it should be something more meaningful such as
```
optionDisplayUseShieldIcon
```
Unless you plan on adding in more images... (in which case a checkbox wouldn't be enough unless you want multiple checkboxes)

Reply:
Should be ok actually for now...

Reply:
I actually plan to add more images so I used a more generic one